app/frontend/src/components/designer/FormSettings.vue

```diff
@@ -1001,6 +1045,7 @@
                                   :href="githubLinkScheduleAndReminderFeature"
                                   class="preview_info_link_field_white"
                                   :target="'_blank'"
+                                  :hreflang="lang"
```

Comment:
An `href` snuck in here (probably from VSCode autocomplete).

Reply:
It is not a vs-code autocomplete. It is an attribute Jordan mentioned that we should add to link for multilanguage support